runtime/etc/Makefile.mli-shared

```diff
@@ -44,11 +44,11 @@ all: buildUserObj buildServer buildClient runTagsCommand FORCE
 
 buildUserObj:
 #
-# Invoke the launcher script, but don't actually link up the launcher binary.
-# Do this by setting CHPL_MAKE_IS_MLI=1.
+# Compile loose source files in CHPLUSEROBJ. Use the TARG_CC for now, but
+# later we may need to roll these into both client and server.
```

Comment:
I would remove everything after the comma and replace it with "as the user code will be run by the server" or something to that effect